nototools/noto_lint.py

```diff
@@ -360,6 +361,27 @@ def printable_font_versions(font):
         version, font_revision)
 
 
+def _build_cmap_dict(filename):
+    data = cmap_data.read_cmap_data_file(filename)
+    script_to_rowdata = cmap_data.create_map_from_table(data.table)
+    return {script: frozenset(tool_utils.parse_int_ranges(rd.ranges))
+            for script, rd in script_to_rowdata.iteritems()}
+
+
+_phase_2_map = None
+_phase_3_map = None
+def _get_cmap_data_for_phase(phase):
+  global _phase_2_map, _phase_3_map
```

Comment:
The indentation throughout most of this file seems to be 4 spaces. Though it differs throughout nototools...